packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeVisualizer.ts

```diff
@@ -81,7 +81,13 @@ function createToolTipContents(schema: SharedTreeSchemaNode): VisualTreeNode {
 }
 
 /**
- * Constructs a VisualTree of the input tree's schema fields in {@link VisualTreeNode} or {@link VisualValueNode}.
+ * Converts the visual represntation from {@link visualizeSharedTreeNodeBySchema} to a visual tree compatible with the devtools-view.
+ * @param tree - the visual representation of the SharedTree.
+ * @returns - the visual representation of type {@link VisualChildNode}
+ *
+ * {@link VisualTreeNode} is omitted when VisualSharedTreeNodeKind is the leaf node, since leaf does not accept object as its child.
```

Comment:
I'm not sure I follow the comment... if we keep it, let's put it under `@remarks` in the TSDoc.